python/cudf/cudf/core/index.py

```diff
@@ -766,22 +802,12 @@ def __getitem__(self, index):
     def as_column(self):
         """Convert the index as a Series.
```

Comment:
Does this return a column or a Series now?

Reply:
Fixed. We return a Column